etcd3/client.py

```diff
@@ -86,34 +74,94 @@ def __init__(self, alarm_type, member_id):
         self.member_id = member_id
 
 
+class Node(object):
+    """Represents an etcd cluster member."""
+
+    creds = None
+    time_retry = 300.0
```

Comment:
Well that's not obvious in the API since I don't expect users to go change every `Node` object that is stored in the `Client` object. And changing a global class is a bad idea.